client/login/magic-login/index.jsx

```diff
@@ -1391,8 +1375,6 @@ class MagicLogin extends Component {
 				<GlobalNotices id="notices" />
 
 				<RequestLoginEmailForm { ...requestLoginEmailFormProps } />
-
-				{ ! shouldShowLoadingEllipsis && this.renderLinks() }
```

Comment:
Was the removal of `this.renderLinks` from here intended? The PR that this reverts only added the condition, so this change goes beyond the scope of a simple reversion.